.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,25 @@
+name: PHP-CS-Fixer
+
+on:
+    pull_request:
+    push:
```

Comment:
```suggestion
on:
    - pull_request
    - push
```

Reply:
This is incorrect YAML notation. I can chage to linear if you want
``` yaml
on: [ pull_request, push ]
```

![image](https://user-images.githubusercontent.com/4967813/141688084-84a2795f-e089-4f53-bebd-7bb45f4708d2.png)

Reply:
the example you tried to apply was without `-`